tutorials/tour/traits.md

```diff
@@ -34,16 +34,20 @@ object TraitsTest extends App {
   val p1 = new Point(2, 3)
   val p2 = new Point(2, 4)
   val p3 = new Point(3, 3)
-  println(p1.isNotSimilar(p2))
-  println(p1.isNotSimilar(p3))
+  val p4 = new Point(2, 3)
+  println(p1.isSimilar(p2))
+  println(p1.isSimilar(p3))
+  // Point's isNotSimilar is defined in Similarity
   println(p1.isNotSimilar(2))
```

Comment:
No, it supposed to illustrate that similarity test fails on `2.isInstanceOf[Point]` test !